host-spec/c04-networking.tm

```diff
@@ -81,551 +128,427 @@
   After a base-layer protocol is established, the Polkadot Host will apply
   the Noise protocol.
 
-  <subsection|Substreams>
+  <subsection|Noise Protocol>
+
+  The Noise protocol is a framework for bulding encryption protocols.
+  <verbatim|libp2p> utilizes that protocol for establishing encrypted
```

Comment:
I'd refer to the libp2p Noise specs, because what is explained in the section below is clearly not enough to re-implement it.